src/assignments/5-loop.js

```diff
@@ -21,6 +37,19 @@ sum=8
 */
 // start here writing the code for program2
 
+
+let numberSum = 687;
+
+function addSum(num){
+
+    let sum = 0;
+    
+
```

Comment:
what the function here? 
you can have a look at the logic how to do this : 
https://stackoverflow.com/questions/38334652/sum-all-the-digits-of-a-number-javascript